packages/libs/client/etc/client.api.md

```diff
@@ -106,6 +106,7 @@ export interface IClient extends IBaseClient {
     dirtyRead: (transaction: IUnsignedCommand) => Promise<ICommandResult>;
     // @deprecated
     getPoll: (transactionDescriptors: ITransactionDescriptor[] | ITransactionDescriptor) => Promise<IPollResponse>;
+    pollOne: (transactionDescriptor: ITransactionDescriptor, options?: IPollOptions) => Promise<ICommandResult>;
```

Comment:
Can you add a message for the changelog to indicate that we also added an alternative to poll that can be used for a single transaction?